src/insert/fixures.ts

```diff
@@ -0,0 +1,85 @@
+import { AD_LABEL_HEIGHT } from 'core/constants';
+import { createAdSlot, wrapSlotInContainer } from 'core/create-ad-slot';
+import { getBreakpoint } from 'lib/detect/detect-breakpoint';
+import { getViewport } from 'lib/detect/detect-viewport';
+import fastdom from '../utils/fastdom-promise';
+import { fillDynamicAdSlot } from './fill-dynamic-advert-slot';
+
+const LARGEST_AD_SIZE = 600; // px, double mpu
+const SPACING = 10; // px, above ad
+
+const insertFootballRightAd = (anchor: HTMLElement) => {
+	const slot = createAdSlot('football-right');
+
+	const container = wrapSlotInContainer(slot, {
+		className: 'ad-slot-container football-right-ad-container',
+	});
+
+	/**
+	 * TODO: Move these to a class in frontend
+	 */
+	container.style.position = 'sticky';
+	container.style.top = '0';
+	container.style.marginTop = '10px';
+
+	void fastdom
+		.mutate(() => {
+			anchor.style.height = '100%';
+			anchor.appendChild(container);
+		})
+		.then(() => fillDynamicAdSlot(slot, false));
+};
+
+const isPageWithAdSpace = () => {
+	const { pageId } = window.guardian.config.page;
+
+	const isFootballPage = pageId.startsWith('football/');
```

Comment:
This might fit better in https://github.com/guardian/commercial/blob/main/src/lib/commercial-features.ts#L61